### ISSUE TITLE
Client / Server Config Differences Cause Issues

### ISSUE THREAD

**Issue by ghost** [zabi94/BringMeTheRings#5]
If the client and server config register different counts for extra slots, errors occur and opening the baubles menu will wreak havoc on the inventory in a similar manner to #1.

It would be nice if on joining a server the mod assumes the extra slots value of the server so there is no issues and need to _manually_ sync configs.

If this is supposed to be an implemented feature already, please let me know so I can provide whatever info may be required!

**Comment by zabi94** [comment]
Unfortunately this is not possible, as inventory slots are hardcoded. I just change the value dynamically at startup, but once the value is set I have no way of modifying it when a player connects to a server

**Comment by zabi94** [comment]
Best I can do is throw an error message when connecting, asking you to change the client config and showing what number is configured on the server

**Comment by ghost** [comment]
Ah, I understand! Limitations are limitations, no getting past that.

It probably would be better to warn the user, considering it can mess up inventories and probably causes other rather unpredictable side effects.

I'm assuming making the inventory slots not hardcoded would be one or both of extremely hacky and a gargantuan effort aha

**Comment by zabi94** [comment]
Yes, it would likely break most baubles. I will try to find some time to add a warning
